util/regexp-assemble/data/942370.data

```diff
@@ -3,22 +3,44 @@
 
 ##!+ i
 
-[\"'`]\s*?\*.+or\W*?[\"'`]\d
-[\"'`]\s*?\*.+xor\W*?[\"'`]\d
-[\"'`]\s*?\*.+div\W*?[\"'`]\d
-[\"'`]\s*?\*.+like\W*?[\"'`]\d
-[\"'`]\s*?\*.+between\W*?[\"'`]\d
-[\"'`]\s*?\*.+and\W*?[\"'`]\d
-[\"'`]\s*?\*.+id\W*?[\"'`]\d
+##!> assemble
+[\"'`]\s*?
+##!=>
+ ##!> assemble
+  ##!> assemble
```

Comment:
You are using two nested assemble blocks here, but you only need one, the outer one doesn't do anything.